packages/now-node/src/index.ts

```diff
@@ -46,18 +66,51 @@ async function downloadInstallAndBundle({
 }
 
 async function compile(
+  workPath: string,
   entrypointPath: string,
   entrypoint: string,
-  config: CompilerConfig
-): Promise<Files> {
+  config: CompilerConfig,
+  { isDev, filesChanged, filesRemoved }: Meta
+): Promise<{ preparedFiles: Files; watch: string[] }> {
   const input = entrypointPath;
   const inputDir = dirname(input);
   const rootIncludeFiles = inputDir.split(sep).pop() || '';
-  const ncc = require('@zeit/ncc');
-  const { code, map, assets } = await ncc(input, {
+  const options: NccOptions = {
     sourceMap: true,
     sourceMapRegister: true,
-  });
+  };
+  let code: string;
+  let map: string | void;
+  let assets: Assets | void;
```

Comment:
```suggestion
  let assets: Assets | undefined;
```

I have never seen void used like this. I thought it could only be used for a function? https://www.typescriptlang.org/docs/handbook/basic-types.html#void